src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,11 @@
+@import "../utils/mixins";
+
+.logo {
```

Comment:
https://prnt.sc/05w4yb6Ogr4o => Add correct href atribute in html, to fix it (It should be "#", not a "/"